frontend/src/Components/Profile/VolunteerProfile.jsx

```diff
@@ -0,0 +1,220 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import axios from 'axios';
+
+import FirstAndLastNameInputs from '../LoginSignup/FirstAndLastNameInputs';
+import EmailPassword from '../LoginSignup/EmailPassword';
+import ProfileTabs from './ProfileTabs';
+import PasswordUpdate from './PasswordUpdate';
+import FileUpload from './FileUpload';
+import SignupVolunteerSubForm from '../LoginSignup/SignupVolunteerSubForm';
+
+
+export default function VolunteerProfile(props) {
+    const {
+        loggedUser,
+        email,
+        password,
+        firstName,
+        lastName,
+        newPassword,
+        confirmPassword,
+        company,
+        title,
+        volunteerSkills,
+        skills,
+        mentor,
+        officeHours,
+        techMockInterview,
+        behavioralMockInterview,
+        professionalSkillsCoach,
+        hostSiteVisit,
+        industrySpeaker,
+    } = props;
+
+    const [ bio, setBio ] = useState(loggedUser.v_bio);
+    const [ linkedIn, setLinkedIn ] = useState(loggedUser.v_linkedin);
+    const [ picFile, setPicFile ] = useState(null);
+
+    const {pathname} = useLocation();
+    const pathName = pathname.split('/');
+
+    const getVolunteerSkills = async () => {
+        try {
+            const { data } = await axios.get(`api/skills/${loggedUser.v_id}`);
+            props.setVolunteerSkills(data.payload);
+        } catch (err) {
+            props.setFeedback(err)
+        }
+    }
+
+    useEffect(() => {
+        props.setVolunteerSkills([1, 3])// getVolunteerSkills();
```

Comment:
did you mean to leave 1, 3 here?

Reply:
removed in next pull.req

Reply:
same thing here, I had to simulate some data since the skills route was not up yet